ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -248,6 +248,15 @@ IE_GPU.onnx_model_gru_fwd_mixed_seq_len_const
 ## Const layer has incorrect dimensions in the output data
 IE_CPU.nothing_to_reverse
 
+# Unsupported dynamic ops
+onnx_size_dyn_op
```

Comment:
Can't we disable this test for IE_CPU only?

Reply:
This manifest file is for IE_CPU and IE_GPU, do you mean this test should be enabled on IE_GPU?
For Interpreter this test case is not skipped.

Reply:
We have discussed it offline